packages/react-i18n/src/test/manager.test.ts

```diff
@@ -612,6 +629,26 @@ describe('Manager', () => {
       await connectionResult.resolve();
     });
 
+    it('does not provide synchronous translations', async () => {
+      const manager = new Manager({...basicDetails, locale: 'en-US'});
+      const connection = new Connection({
+        id: createID(),
+        translations: getTranslation,
+      });
+
+      // Connected, but not resolved
+      await manager
+        .connect(
+          connection,
+          noop,
+        )
+        .resolve();
+
+      const translationsByID = manager.extract();
+      // @ts-ignore (Object.values)
```

Comment:
why this?

Reply:
It was here before but I could get rid of it without issue /shrug